.github/workflows/ci.yaml

```diff
@@ -0,0 +1,65 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+name: ci
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [10, 12, 13]
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
@JustinBeckwith any reason you went with v1?

Reply:
Ahhhhh, this is potentially good feedback for @ethomson :) Honestly I have a CI script I wrote  a few months ago, and I mostly copy it around from project to project.  I have no way to know (that I know of) when dependencies of actions are updated, and when I should bump them.  For things like `package.json`, I can set up renovatebot or dependabot.  For CI scripts ... I am kind of left guessing. 

Happy to try v2, and thank you for being vigilant!

Reply:
This is good feedback - I appreciate it.  I'll give this some thought.  Cheers!